includes/core/classes/class-user.php

```diff
@@ -72,6 +72,10 @@ protected function setup_hooks(): void {
 	public function profile_fields( WP_User $user ): void {
 		$event_updates_opt_in = get_user_meta( $user->ID, 'gp-event-updates-opt-in', true );
 
+		$default_tz   = static::get_system_timezone();
```

Comment:
Shouldn't this be
`$timezone_default = get_option( 'timezone_string' );`?

(reversed in var to match the other two format below)

Reply:
Ok never mind, though I think we can move this to method to `Utility` class in another PR.